### DISCUSSION TITLE
Maximum of 3600 seconds

### DISCUSSION THREAD

**Discussion by FugiOnline** [Djelibeybi/homebridge-delayed-occupancy#6] [Q&A]
Is there a technical reason for the maximum of 3600 seconds? I'm looking to add an occupancy delay of 24h which is triggered when my system detects rain. I.e. the occupancy goes off if there is no rain for the last 24h.

Cheers

**Comment by Djelibeybi** [comment]
To be honest, I don't know. My fork was really only created to fix a few things that were bugging me specifically. 😁 

I recommend using https://github.com/Jason-Morcos/homebridge-magic-occupancy instead, which is Homebridge Verified and also allows up to 24 hours by default.

**Reply by FugiOnline** [reply]
Check. Great alternative, I'll look into it. Thanks!

**Reply by Djelibeybi** [reply]
You're welcome!
